src/author.rs

```diff
@@ -13,25 +13,16 @@ pub struct Author {
     name: String,
 }
 
-/// The methods and associated functions on and for an `Author` instance,
-/// respectively.
+/// The methods and associated functions on and for an `Author`, respectively.
 impl Author {
     /// The getter method for the field `email` of the corresponding struct.
-    ///
-    /// Since the struct `Author` defines its field `email` to be private, this
-    /// getter is used in order to operate on this field.  It returns a
-    /// read-only deep copy of the content.
-    pub fn email(&self) -> String {
-        String::from(&self.email)
+    pub fn email(&self) -> &str {
+        &self.email
     }
 
     /// The getter method for the field `name` of the corresponding struct.
-    ///
-    /// Since the struct `Author` defines its field `name` to be private, this
-    /// getter is used in order to operate on this field.  It returns a
-    /// read-only deep copy of the content.
-    pub fn name(&self) -> String {
-        String::from(&self.name)
+    pub fn name(&self) -> &str {
+        &self.name
```

Comment:
This should be squashed with the commit, that introduced these as functions returning `String`.